test/emf/emf_container_test.go

```diff
@@ -29,37 +31,51 @@ func (t *EMFECSTestRunner) Validate() status.TestGroupResult {
 	}
 }
 
-func (t *EMFECSTestRunner) GetTestName() string {
-	return "EMFOnECS"
+func (t *EMFTestRunner) GetTestName() string {
+	return "EMF Container Tests"
 
 }
 
-func (t *EMFECSTestRunner) GetAgentConfigFileName() string {
+func (t *EMFTestRunner) GetAgentConfigFileName() string {
 	return "./resources/config.json"
 }
 
-func (t *EMFECSTestRunner) GetAgentRunDuration() time.Duration {
+func (t *EMFTestRunner) GetAgentRunDuration() time.Duration {
 	return 3 * time.Minute
 }
 
-func (t *EMFECSTestRunner) GetMeasuredMetrics() []string {
+func (t *EMFTestRunner) GetMeasuredMetrics() []string {
 	return []string{"EMFCounter"}
 }
 
-func (t *EMFECSTestRunner) validateEMFOnECSMetrics(metricName string) status.TestResult {
+func (t *EMFTestRunner) validateEMFMetrics(metricName string) status.TestResult {
+	namespace := ""
+	var dims []types.Dimension
+	var failed []dimension.Instruction
+	if t.testName == "EMF_ECS" {
+		namespace = "EMFNameSpace"
```

Comment:
Please update namespace by following the format of `[feature]/[infra]` e.g. EMF/ECS

Reply:
updated namespace in latest commit